.github/workflows/restore-pg-backup.yml

```diff
@@ -0,0 +1,48 @@
+name: Restore from backup
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: Environment to restore
+        required: true
+        type: choice
+        default: qa
+        options:
+        - qa
+      backup-file:
+        description: File to be restored
+        required: true
+      compression:
+        description: Is the backup file compressed
+        type: boolean
+        required: false
+        default: false
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set variables
+        run: |
+          source global_config/qa.sh
+          # source global_config/${{ github.event.inputs.environment }}.sh
+          echo "STORAGE_ACCOUNT=${RESOURCE_NAME_PREFIX}rttdbbkp${CONFIG_SHORT}sa" >> $GITHUB_ENV
+          echo "RESOURCE_GROUP=${RESOURCE_NAME_PREFIX}-rtt-${CONFIG_SHORT}-rg" >> $GITHUB_ENV
+          echo "APP_NAME=register-${CONFIG}" >> $GITHUB_ENV
+          echo "APP_ENV=${CONFIG}" >> $GITHUB_ENV
```

Comment:
Is this used?

Reply:
removed APP_ENV, as was just used in testing